src/Application.php

```diff
@@ -47,15 +65,12 @@ public function __construct($app_root, $env)
         $this->dirs = $this->getDefaultDirs();
     }
 
-    /**
-     * @return Container
-     */
-    public function getContainer()
+    public function getContainer(): ?Container
```

Comment:
Why nullable ?
The original `@return Container` indicates this should not be null.